src/View/Helper/CredentialsHelper.php

```diff
@@ -50,7 +50,7 @@ public function filter(mixed $in): mixed
             return $in;
         }
         preg_match_all($regexp, $in, $tokens);
-        if (empty($tokens[0])) {
+        if (count($tokens[0]) === 0) {
```

Comment:
```suggestion
        if ($tokens[0] === []) {
```
If the index is always going to be set.

Reply:
For the above regex it will always be set.